terraspace/app/stacks/event/secrets.tf

```diff
@@ -0,0 +1,21 @@
+resource "aws_kms_key" "event_stack" {
+  description = var.secrets_key.description
+}
+
+resource "aws_kms_alias" "event_stack" {
+  name          = "alias/${var.secrets_key.name}"
+  target_key_id = aws_kms_key.event_stack.key_id
+}
```

Comment:
you might want to consider that these two resources should live outside of the `event` stack as they could be reused by other components.

Reply:
We could put that in the shared stack to have a single key for all Elastic IPFS components, but I don't really see benefit in doing that (except having less resources to maintain and therefore reducing complexity). With the "one key per stack" strategy we have better decoupling and security responsibility/concern.